DOCS-16652 Adds Reversing State (#282)
mongodb/docs-cluster-to-cluster-sync: source/reference/mongosync-states.txt

     - - ``GET`` :ref:`/progress <c2c-api-progress>`

>        - ``GET`` :ref:`/reverse <c2c-api-reverse>`

Reviewer: ```suggestion
       - ``POST`` :ref:`/reverse <c2c-api-reverse>`
```
reverse is a `POST` command.

Author: Fixed

   * - .. _c2c-state-reversing:

       ``REVERSING``

     - The sync process copies metadata from the destination
       cluster to the source cluster. Then, MongoDB swaps the
       source and destination clusters and resumes applying
       change events.

>      - - ``POST`` :ref:`/reverse <c2c-api-reverse>`
>        - ``GET`` :ref:`/progress <c2c-api-progress>`

Reviewer: ```suggestion
     - - ``GET`` :ref:`/progress <c2c-api-progress>`
```

I just noticed `reverse` is not a possible command under this state.

Author: Fixed

     - - ``GET`` :ref:`/progress <c2c-api-progress>`
       - ``GET`` :ref:`/reverse <c2c-api-reverse>`

   * - .. _c2c-state-reversing:

       ``REVERSING``

>      - The sync process copies metadata from the destination
>        cluster to the source cluster. When complete, the phase
>        reverts to applying change events and the original source
>        cluster becomes the new destination cluster while the old
>        destination becomes the new source.

Reviewer 2: Not sure about the use of "revert" here. Would it be accurate to say something like:

```suggestion
     - The sync process copies metadata from the destination
       cluster to the source cluster. Then, MongoDB swaps the
       source and destination clusters and resumes applying
       change events.
```

I don't love the use of "swap" in my example from a technical standpoint, but if possible I'd prefer a shorter way of saying that source and destination switch places.
